ODBP.Server/Features/Sitemap/SitemapIndex/SitemapIndexModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Globalization;
+using System.Xml.Serialization;
+
+namespace ODBP.Features.Sitemap.SitemapIndex;
+
+[XmlRoot(ElementName = "urlset", Namespace = "http://www.sitemaps.org/schemas/sitemap/0.9")]
+public class SitemapIndexModel
+{
+    [XmlElement("sitemap")]
+    public required List<SitemapLink> Sitemaps { get; set; }
+}
+
+public class SitemapLink
+{
+    [XmlElement("loc")]
+    public required string Loc { get; set; }
+
+    [XmlElement("lastmod")]
+    public string LastMod { get; set; } = DateTimeOffset.Now.ToString("o", CultureInfo.InvariantCulture);
```

Comment:
da sitemaps schema verwijst naar een ander format?

Reply:
wij maken deze sitemap niet voor google. is ergens gespecificeerd waar hij aan moet voldoen? het lijkt me goed om daar vanuit code of documentatie naar te verwijzen

Reply:
Het schema waar dit veld uit komt is het [sitemap schema](https://www.sitemaps.org/protocol.html). Daar staat dat de lastmod moet voldoen aan [W3C Datetime](http://www.w3.org/TR/NOTE-datetime). Daar voldoen we aan. Ik zal een xml comment toevoegen aan de lastmod velden.